leveldb/leveldb_test.go

```
@@ -0,0 +1,287 @@
+	require.Nil(t, err)
+
+}
+
+func Test_Reset(t *testing.T) {
+	db := New()
+
+	db.Set([]byte("key1"), []byte("value1"), time.Second*1)
+	db.Set([]byte("key2"), []byte("value2"), time.Second*1)
+	db.Set([]byte("key3"), []byte("value3"), time.Second*1)
+
+	db.Reset()
```

golangci-lint (1.62.2)

Error return value of `db.Reset` is not checked

(errcheck)